io/disk/disktest.py.data/README.txt

```diff
@@ -8,6 +8,7 @@ each disk) and checking failures after each chunk is written.
 Available parameters
 --------------------
 
-disk       - Disk to be used in test.
+disk       - Provide the test disk name /dev/sda or /dev/mapper/mpatha or 
+             scsi-360050768108000000000283 or nvme-eui.364555305250000003-part1
```

Comment:
Add the command here, from where can we get this details,

Reply:
done